README.md

```diff
@@ -59,14 +59,10 @@ This will start a local development server on port 5000 (http://localhost:5000).
 
 ## Adding a project
 
-Each of the projects is a file in the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
-folder - to add a new one, create a new file named after the project, ending in `.yaml`. Ensure all spaces and special
-characters are replaced with `-`. Make sure too that the project hasn't already been inside the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
-folder
-
 Codetribute can read from both GitHub and Bugzilla. To add a new entry to the site, create a file `<project-name>.yml` 
-in `src/data` using the [template example](#template-example) as the initial setup. For inspiration, check out the 
-[Taskcluster](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/data/taskcluster.yaml) yaml file.
+in `src/data` using the [template example](#template-example) as the initial setup. Ensure all spaces and special
+characters are replaced with `-`. Make sure that the project is not already in the `src/data` folder.
```

Comment:
Remove "Ensure all spaces and special characters are replaced with -. Make sure that the project is not already in the src/data folder."